action/test/parse/pre-release.js

```diff
@@ -11,12 +11,12 @@ tap.test('parse -> pre-release -> direct match', async assert => {
 
   sinon.stub(core, 'info')
 
-  const proceed = parse('chore(deps): bump api-problem from 6.1.2 to 6.1.4-prerelease in /path', 'preminor')
+  const proceed = parse('chore(deps): bump api-problem from 6.1.2 to 6.1.4-prerelease in /path', [], 'preminor')
```

Comment:
This test is flawed. Neither of the prerelease versions are defined in the weights object in `parse.js`. So a premajor update will also be merged, even if the highest auto-merge target is set to preminor.

I'll change it so that premajor has a higher target than major etc.

Reply:
👍